src/WinGetUtil/WinGetUtil.h

```diff
@@ -129,4 +129,46 @@ extern "C"
         WINGET_STRING versionA,
         WINGET_STRING versionB,
         INT* comparisonResult);
+
+    // A handle to the metadata collection object.
+    typedef void* WINGET_INSTALLER_METADATA_COLLECTION_HANDLE;

```

Comment:
>typedef void* WINGET_INSTALLER_METADATA_COLLECTION_HANDLE; [](http://example.com/codeflow?start=4&length=58)

nit: should we put all typedefs in the top?

Reply:
Should we?  I do like things being defined close to where they are used.  If the type is being used in a broad fashion, the top makes sense to me.  But if it is used narrowly, then immediately above the use feels better.